tests/shutdown/shutdown.pm

```diff
@@ -29,4 +30,9 @@ sub test_flags {
     return {fatal => 1};
 }
 
+sub post_fail_hook {
+    my ($self) = shift;
+    $self->SUPER::post_fail_hook;
```

Comment:
what's the point of this? You are just calling the post_fail_hook of the parent. That should be the same as not defining an override here. Did you miss to commit something?

Reply:
yes. remove this.